apps/frontend/src/app/validators/map-edit.validator.ts

```diff
@@ -0,0 +1,10 @@
+import { FormGroup } from '@angular/forms';
+
+export function atLeastOneFileRequired(fieldNames: string[]) {
```

Comment:
Validator is good but isn't specific to files. Also may as well make it more flexible, use a `filter(...).length` and have it be `atLeastNRequired(numRequired: number, fieldNames: string[])`